Library/Homebrew/mktemp.rb

```diff
@@ -28,6 +29,11 @@ def retain?
     @retain
   end
 
+  # True if the source files should be retained.
+  def retain_in_sources?
```

Comment:
```suggestion
  def retain_in_cache?
```

Reply:
Done in [676e3d4](https://github.com/Homebrew/brew/pull/13608/commits/676e3d4923c3c92c46f479c54e4cbf2ac0fb71c0)